en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        self.parseClassName = "Todo"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = true
+        self.objectsPerPage = 25
+    }
+
+    required init!(coder aDecoder: NSCoder!) {
+        super.init(coder: aDecoder)
+        self.parseClassName = "Todo"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = true
+        self.objectsPerPage = 25
+    }
+    
+    override func queryForTable() -> PFQuery {
+        var query = PFQuery(className: self.parseClassName!)
+        
+        // If no objects are loaded in memory, we look to the cache first to fill the table
+        // and then subsequently do a query against the network.
+        if self.objects!.count == 0 {
+            query.cachePolicy = .CacheThenNetwork
+        }
+        
+        query.orderByDescending("createdAt")
+        
+        return query
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath, object: PFObject?) -> PFTableViewCell? {
+        let cellIdentifier = "cell"
+        
+        var cell = tableView.dequeueReusableCellWithIdentifier(cellIdentifier) as? PFTableViewCell
+        if cell == nil {
+            cell = PFTableViewCell(style: .Subtitle, reuseIdentifier: cellIdentifier)
+        }
+        
+        // Configure the cell to show todo item with a priority at the bottom
+        if let object = object { 
+            cell!.textLabel!.text = object["text"] as? String
```

Comment:
Okay, I was thinking about something definitely too complicated!

Nevertheless  `cell` is definitely not going to be `nil` and maybe we want to unwrap it with the exclamation mark as Apple suggests:
>[...]because it can be safely assumed to have a value all of the time.

>These kinds of optionals are defined as implicitly unwrapped optionals. You write an implicitly unwrapped optional by placing an exclamation mark (String!) rather than a question mark (String?) after the type that you want to make optional.

So we will have this:
```swift
override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath, object: PFObject?) -> PFTableViewCell? {
  let cellIdentifier = "cell"

  var cell = tableView.dequeueReusableCellWithIdentifier(cellIdentifier) as? PFTableViewCell
  if cell == nil {
    cell = PFTableViewCell(style: .Subtitle, reuseIdentifier: cellIdentifier)
  }

  // Configure the cell to show todo item with a priority at the bottom
  if let object = object { 
    cell!.textLabel?.text = object["text"] as? String
    let priority = object["priority"] as? String
    cell!.detailTextLabel?.text = "Priority \(priority)"
  }
  return cell
}
```